akka-projection-core/src/main/scala/akka/projection/ProjectionId.scala

```diff
@@ -82,10 +82,7 @@ object ProjectionId {
    * @return an [[java.util.Set]] of [[ProjectionId]]s
    */
   def of(name: String, keys: java.util.Set[String]): java.util.Set[ProjectionId] =
-    keys
-      .stream()
-      .map(key => new ProjectionId(name, key))
-      .collect(Collectors.toSet())
+    keys.asScala.map { key: String => new ProjectionId(name, key) }.asJava
```

Comment:
I'm wondering is we need this method. We can leave it for now and reassess when we go back to the ShardedDaemonProcess integration.

Reply:
why change it to Scala?

Reply:
I will give it another try. The call `map(key => new ProjectionId(name, key))` wasn't compiling in Scala 2.12. I don't remember what was the reason.

Reply:
this is the error we get in 2.12

```scala
[error]  found   : java.util.function.Function[String,akka.projection.ProjectionId]
[error]  required: java.util.function.Function[_ >: String, _ <: R]
[error]       .map(key => new ProjectionId(name, key))

```

I tried to solve this by using 
`import scala.compat.java8.FunctionConverters._` and different encoding alternatives to see if the compiler would understand what I wanted, but nothing worked out. So I kind of give up and decided to use scala collections instead